cmd/tap-fitter/main.go

```diff
@@ -16,6 +16,7 @@ type TapFitterTemplate struct {
 	compositePath         string
 	catalogPath           string
 	compositeSpec         *composite.Template
+	writer                io.Writer
```

Comment:
I didn't see where this was initialized.  Are we depending on default initialization behavior?

Reply:
Still don't see where this is initialized.